### ISSUE TITLE
[Quest] Into realm of shadow

### ISSUE THREAD

**Issue by HannibalRoG** [AshamaneProject/AshamaneCore#772]
**Description:**
The quest is autocomplete, I have reviewed all the information in the DB and I have not found anything that makes it autocomplete.

**Usefull Wowhead links:**

http://es.wowhead.com/quest=12687/en-el-reino-de-las-sombras

@Neredun can you take a look here.

**Comment by Neredun** [comment]
I just checked this on the quest_template and it's normal (flag isn't set as Autocompletable) But the quest itself doesn't have objectives in Quest_objective (and his _locale version too). At least in my db.

**Comment by HannibalRoG** [comment]
Yes, but in trinity has the same state and it is not autocomplete.

**Comment by Neredun** [comment]
Just tested it in a new fresh compile of Trinity. And it was autocompleted
![wowscrnshot_040418_032325](https://user-images.githubusercontent.com/21986277/38291811-bf192592-37b7-11e8-96a2-4c1ff2e75d7b.jpg)
![wowscrnshot_040418_032318](https://user-images.githubusercontent.com/21986277/38291817-c3c985f0-37b7-11e8-8d9c-22a6bacea3c0.jpg)

**Comment by HannibalRoG** [comment]
Well, i think it need quest objetives.

**Comment by Neredun** [comment]
i remember well that it was only one objective and it was completed when you cast http://www.wowhead.com/spell=52362 on Death's Breach. After beating the Dark Riders of Acherus on the Realms of Shadows (http://www.wowhead.com/npc=28768) and mounting the Acherus Deathcharger who have that spell. (http://www.wowhead.com/npc=28782/) I think it's worth to mention that i already phased the NPCs from the Realm of Shadows Version so that would be the only thing left to do with this quest
